packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -7,13 +7,16 @@ import { IconButton } from '../IconButton/IconButton';
 import { HorizontalGroup } from '../Layout/Layout';
 import panelArtDark from './panelArt_dark.svg';
 import panelArtLight from './panelArt_light.svg';
+import { AlertVariant } from '..';
```

Comment:
Does this shorthand import work in builds as well? I remember there were some issues with those but could be wrong.